app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="action_add_to_playlist">Voeg toe aan afspeellijst...</string>
+    <string name="action_delete">Verwijder</string>
+    <string name="action_edit_smart_playlist">Bewerken...</string>
+    <string name="action_skip">Overslaan</string>
+    <string name="action_play">Afspelen</string>
+    <string name="action_pause">Pauzeer</string>
+    <string name="action_undo">Ongedaan maken</string>
+    <string name="action_create">Creëren</string>
+    <string name="action_cancel">Annuleren</string>
+    <string name="action_add">Toevoegen</string>
+    <string name="action_add_new">Voeg een nieuwe toe</string>
+    <string name="action_agree">Overeenkomen</string>
+    <string name="action_try_again">Probeer opnieuw</string>
+    <string name="action_open_settings">Open instellingen</string>
+    <string name="action_make_new_playlist">Maak een nieuwe afspeellijst</string>
+    <string name="action_add_rule">Regel toevoegen</string>
+    <string name="action_save">Opslaan</string>
+    <string name="action_discard">Annuleren</string>
+    <string name="action_discard_changes">Wijzigingen ongedaan maken</string>
+    <string name="action_edit_playlist_rules">Bewerk filters</string>
+    <string name="action_shuffle_all">Shuffle alles</string>
+    <string name="confirm_disable_shuffle">Shuffle uitgeschakeld</string>
+    <string name="confirm_enable_shuffle">Shuffle ingeschakeld</string>
+    <string name="confirm_disable_repeat">Herhalen uitgeschakeld</string>
+    <string name="confirm_enable_repeat">Herhalen ingeschakeld</string>
+    <string name="confirm_enable_multi_repeat">Multi-herhalen (%d) ingeschakeld</string>
+    <string name="confirm_enable_repeat_one">Één keer herhalen ingeschakeld</string>
+    <string name="confirm_disable_sleep_timer">Slaaptimer uitgeschakeld</string>
+    <string name="confirm_refresh_library">Bibliotheek herladen gelukt</string>
+    <string name="confirm_clear_queue">Wachtrij gewist</string>
+    <string name="action_previous">Vorige</string>
+    <string name="action_navigate_menu">Navigeer naar...</string>
+    <string name="action_exclude">Map uitsluiten</string>
+    <string name="action_include">Map insluiten</string>
+    <string name="action_set_sleep_timer">Slaaptimer...</string>
+    <string name="action_export_playlist">Wachtrij opslaan als afspeellijst...</string>
+    <string name="widget_compact_name">Jockey Compact</string>
+    <string name="widget_wide_name">Jockey Wijd</string>
+    <string name="widget_square_name">Jockey Illustratie</string>
+    <string name="confirm_removed_rule">Regel verwijderd</string>
+    <string name="prompt_save_changes">Wijzigingen opslaan?</string>
+    <string name="prompt_discard_changes">Wijzigingen ongedaan maken?</string>
+    <string name="confirm_application_exit">Druk nog een keer om af te sluiten</string>
+    <string name="message_play_error_not_found">Jockey kon “%1$s” niet vinden</string>
```

Comment:
This probably needs to use the HTML entity code for the curly double-quotes:  `&#8220;%1$s&#8221;` Same thing in several strings after this one.

Reply:
Fixed.